core/src/test/scala/unit/kafka/server/ReplicaManagerConcurrencyTest.scala

```diff
@@ -50,12 +51,15 @@ import scala.collection.{immutable, mutable}
 import scala.jdk.CollectionConverters._
 import scala.util.Random
 
-class ReplicaManagerConcurrencyTest {
+class ReplicaManagerConcurrencyTest extends Logging {
 
   private val time = new MockTime()
```

Comment:
should we close time.scheduler.close() as well?

Reply:
Nice! Updated.